tools/cli/commands/start-demo.js

```diff
@@ -25,17 +25,19 @@ function startDemo() {
         },
     });
 
-    const server = new WebpackDevServer(compiler, {
-        contentBase: path.join(pathToDemo),
-        publicPath: `http://localhost:${PORT}/`,
-        watchContentBase: true,
-        port: PORT,
-        hot: true,
-        disableHostCheck: true,
-        watchOptions: {
-            poll: 500,
+    const server = new WebpackDevServer({
+        static: {
+            directory: path.join(pathToDemo),
+            publicPath: `http://localhost:${PORT}/`,
+            watch: {
+                usePolling: true,
+                pollInterval: 500,
+            },
         },
-    });
+        allowedHosts: 'all',
+        hot: true,
+        port: PORT,
+    }, compiler);
 
 
     server.listen(PORT, '127.0.0.1', () => {

```

Comment:
![image](https://user-images.githubusercontent.com/21058220/225347046-73bb915d-9131-4524-b2d1-6f1f9b92a345.png)
Following the deprication mention about `listen`, this is what I've done - added `host: 'localhost',` and changed the server start command following the guide (+ you can add `/* eslint-disable max-lines-per-function */` above the function):

```js
    const server = new WebpackDevServer({
        static: {
            directory: path.join(pathToDemo),
            publicPath: `http://localhost:${PORT}/`,
            watch: {
                usePolling: true,
                pollInterval: 500,
            },
        },
        allowedHosts: 'all',
        hot: true,
        host: 'localhost',
        port: PORT,
    }, compiler);

    (async () => {
        server.startCallback(() => {
            console.log(`Starting server on http://localhost:${PORT}`);
        });
    })();
```

**Note:** omitting the `host` property will also host it on http://`192.168.**.**`:3000/